src/trait_manager/view/TraitColorView.js

```diff
@@ -10,16 +10,17 @@ module.exports = TraitView.extend({
   getInputEl() {
     if (!this.$input) {
       var value = this.getModelValue();
-      var inputNumber = new InputColor({
+      var inputColor = new InputColor({
         target: this.config.em,
         contClass: this.ppfx + 'field-color',
         model: this.model,
         ppfx: this.ppfx
       });
-      this.input = inputNumber.render();
+      this.input = inputColor.render();
       this.$input = this.input.colorEl;
       value = value || '';
       this.model.set('value', value).trigger('change:value');
+      this.input.setValue(value);
```

Comment:
Hey @artf,

Small followup from my last pull request. I just noticed that the input color value is always initially "undefined" after the InputColor is created. Setting the value here into the input here seems to resolve it.

Reply:
Good catch 👍